docs/source/quick.rst

```diff
@@ -0,0 +1,224 @@
+###########
+Quick Start
+###########
+
+This is a quick start guide to get something up quickly to see the API in
+action.
+
+Installing
+==========
+
+We'll simply clone the git tree and use Custodia in place to quickly test it.
+Clone the repository, and run the make egg_info command to prepare the tree for
+execution::
+   $ git clone https://github.com/latchset/custodia.git
+   $ cd custodia
+   $ make egg_info
+
+Configuring
+===========
+
+We'll use a simple a bare minimum configuration to start off.
+
+Write a file named quick.conf with the followin contents (feel free to omit the
+comments)::
+
+   [global]
+   # Listen on a socket named './quick'
+   server_socket = ./quick
+
+   # Accepts any request that specifies an arbitrary REMOTE_USER header
+   [auth:header]
+   handler = SimpleHeaderAuth
+   header = REMOTE_USER
+
+   # Allow requests for all paths under '/' and '/secrets/'
+   [authz:paths]
+   handler = SimplePathAuthz
+   paths = / /secrets/
+
+   # Store secrets in a sqlite database called quick.db in the table 'secrets'
+   [store:quick]
+   handler = SqliteStore
+   dburi = quick.db
+   table = secrets
+
+   # Serve starting from '/' and using the 'quick' store and the 'Root' handler
+   [/]
+   handler = Root
+   store = quick
+
+
+Running
+=======
+
+Now all we need is to start the server.
+We do that with the following command::
+   $ python -m custodia.server quick.conf
+
+The server will output to the terminal logs about the operations being peformed
+against it.
+
+
+Testing
+=======
+
+Once the server is started we can move to another terminal to test it.
+To avoid some typing we'll create a shell alias::
+   $ alias curls='curl -s --unix-socket ./quick -H "REMOTE_USER: me"'
+
+Let's use curl to fetch the root object
+::
+   $ curls http://localhost/
+   {"message": "Quis custodiet ipsos custodes?"}
+
+The message "Quis custodiet ipsos custodes?" is emitted by the Root handler by
+default when querying the root ('/'). It is used to test that the basic
+pipeline is working and authorizing correctly.
+
+The Root handler automatically adds also a 'secrets' handler under the path
+'/secrets/', this is the actual basename of our secrets storage and will always
+be used going forward.
+
+Let's now create a new container for our secrets
+::
+   $ curls -X PUT http://localhost/secrets/bucket/
```

Comment:
This `PUT` requests doesn't work properly, based on the docs, to create a container we should use a `POST` request: https://custodia.readthedocs.io/en/latest/api.html#creating-containers